### ISSUE TITLE
[Bug]double right-click on Macbook trackpad does require a bit of finesse

### ISSUE THREAD

**Issue by DennisVerez** [LiveEnhancementSuite/LESforMacOS#36]
**Describe the bug**
double right-click on Macbook trackpad does require a bit of finesse

**To Reproduce**
Trackpad right-click works perfectly in Ableton, however, the time between the 1st and the 2nd right-click to get the menu has to be just right. It works, but man, what a pain.

**Expected behaviour**
2x right click should just work with a trackpad (right?)

**Screenshots**
-

**Desktop (please complete the following information):**
2021 MacBook Pro m1 
Ventura 13.5.2
Ableton 11.3.10
LES v1.3.3-alpha (https://github.com/LiveEnhancementSuite/LESforMacOS/releases/tag/v1.3.3)

**Additional context**
I saw a feature request that maybe very helpful, like opening LES menu with a key+click (option+click?)

**Comment by SolBeys** [comment]
Yes it's a problem for me as well, you need to click 2x slowly to open the menu everytime.
